### PR TITLE
Landing page logo and header adjustments

### PR DESCRIPTION
**Pull request by timoballard**
[Link to ZenHub issue.](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/848)

## What does this change?
Adjusts DOJ logo and title text styling according to the November design audit findings.

## Screenshots (for front-end PR):
https://user-images.githubusercontent.com/1425377/104355261-06dcda80-54d0-11eb-97d5-5a47bc044eaa.mov

### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for, document, and establish a testing plan for any behavior that may vary across environments or is otherwise difficult to test.
+ [x] Check for [accessibility](/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for any behavior that may vary across environments or is difficult to test, and ensure that it is well-understood, documented, and that there is a testing plan in place.
+ [x] Re-check for [accessibility](/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

**Comment by Jkrzy** [comment]
Thanks @timoballard looks good to me! Please complete the PR checklist if you ran through those steps already.

**Comment by cantsin** [comment]
Looks good on IE11/Edge as well.

File: crt_portal/static/sass/custom/landing.scss [hunk 1]
```diff
@@ -161,19 +161,34 @@ $arrow-height: 24px;
...
+  }
+
   .crt-landing--logo_subheading {
     @extend .usa-footer__logo-subheading;
-    font-size: $theme-text-font-size-sm;
-    @media screen and (max-width: 400px) {
+    font-size: $theme-text-font-size-xs;
+    line-height: 2.25;
+    margin-top: 0.25rem;
+    @media screen and (max-width: 480px) {
```
Comment (cantsin) [inline crt_portal/static/sass/custom/landing.scss:178]: If we're changing back to `480px`, we can use `mobile-lg` as that's shorthand for this responsive width. Not sure why it was `400px` offhand... I think I wrote `400px` but I can't remember why :sweat_smile:

**Review by cantsin — CHANGES_REQUESTED** [review]
Generally LGTM, one tweak suggested

**Review by cantsin — APPROVED** [review]
LGTM!

**Pull request merged**